pros/serial/terminal/terminal.py

```diff
@@ -224,7 +231,75 @@ def reader(self):
                 if data[0] == b'sout':
                     text = decode_bytes_to_str(data[1])
                 elif data[0] == b'serr':
-                    text = '{}{}{}'.format(colorama.Fore.RED, decode_bytes_to_str(data[1]), colorama.Style.RESET_ALL)
+                    # print(len(text))
+                    addr = "0x" + decode_bytes_to_str(data[1])[:7]
+
+                    convert_trace = (self.beginStackTrace and
+                                     addr.isalnum() and
+                                     addr[3] != 'x' and
+                                     self.convert_stack_traces and
+                                     ((os.name != 'nt') or os.environ.get('PROS_TOOLCHAIN')))
+
+                    if convert_trace:
+                        if os.name == 'nt' and os.environ.get('PROS_TOOLCHAIN'):
+                            addr2line_path = os.path.join(os.environ.get('PROS_TOOLCHAIN'), 'bin',
+                                                          'arm-none-eabi-addr2line')
+                        else:
+                            addr2line_path = 'arm-none-eabi-addr2line'
+
+                        def getTrace(s, path):
+                            if not os.path.exists(path):
+                                return ''
+                            temp = subprocess.check_output([addr2line_path, '-faps', '-e', path, s]).decode('utf-8')
```

Comment:
```suggestion
                            temp = subprocess.check_output([addr2line_path, '-faps', '-e', '-C', path, s]).decode('utf-8')
```
Right now C++ function names look like letter soup, since they're not demangled.